src/feeds/account-friendships.feed.ts

```diff
@@ -0,0 +1,36 @@
+import { Expose, plainToClassFromExist } from 'class-transformer';
+import { Feed } from '../core/feed';
+import { AccountFriendshipsFeedResponse, AccountFriendshipsFeedResponseUsersItem } from '../responses';
+
+export class AccountFriendshipsFeed extends Feed<
```

Comment:
Such. A. Nice. Code.
Everything is perfect except naming convention :)
This class should be called `PendingFriendships` or `FriendshipsPending` (at my glance first one is better) respective to `friendships/pending` endpoint. And all the consequences should be renamed too.